python/kserve/poetry.lock

```diff
@@ -2633,10 +2668,91 @@ serve = ["aiohttp (>=3.7)", "aiohttp-cors", "aiorwlock", "colorful", "fastapi",
 train = ["pandas", "requests", "tabulate", "tensorboardX (>=1.9)"]
 tune = ["pandas", "requests", "tabulate", "tensorboardX (>=1.9)"]
 
+[[package]]
+name = "ray"
+version = "2.6.1"
```

Comment:
can you temporarily revert/pin `ray` to version `2.4.0` to find out if that will allow the test_dataplane` unit test to pass?

Reply:
Done

Reply:
The test are passing with `ray` version `2.4.0`.